docs/data/joy/customization/theme-typography/CustomFontFamily.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { CssVarsProvider, extendTheme } from '@mui/joy/styles';
+import CssBaseline from '@mui/joy/CssBaseline';
+import Typography from '@mui/joy/Typography';
+
+const customTheme = extendTheme({
+  fontFamily: {
+    body: 'SF Pro Text',
+  },
+  cssVarPrefix: 'font',
```

Comment:
@siriwatknp Unless and untill custom `cssVarPrefix` is added, cutom font is not applying. any other way to not use custom `cssVarPrefix` and still apply custom font family?

Reply:
Hmm, that's weird. `cssVarPrefix` should not be required.

Reply:
It seems working, see https://codesandbox.io/s/gifted-silence-z9jvrl?file=/index.tsx

Reply:
i'm seeing `something wrong` screen on opening above link